circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  python:
+    version: 2.7.10
```

Comment:
We use `2.7.11` but this is preinstalled on their build nodes and cuts our build time by ~40s